app/src/main/java/tech/relaycorp/letro/onboarding/registration/RegistrationDomainProvider.kt

```diff
@@ -5,21 +5,34 @@ import javax.inject.Inject
 
 interface RegistrationDomainProvider {
     fun getDomain(): String
+    fun getDomainLocale(): Locale
 }
 
 class RegistrationDomainProviderImpl @Inject constructor() : RegistrationDomainProvider {
 
+    private val lazyDomainLocale: Locale by lazy {
+        Locale.getDefault()
+    }
+
     private val lazyDomain: String by lazy {
         val locale = Locale.getDefault()
-        when (locale.toString()) {
-            "en_GB" -> "@cuppa.fans"
-            "en_US" -> "@applepie.rocks"
-            "es_VE" -> "@guarapo.cafe"
-            else -> "@nautilus.ink"
-        }
+        DOMAIN_BY_LOCALE[locale.toString()] ?: FALLBACK_DOMAIN
     }
 
     override fun getDomain(): String {
         return lazyDomain
     }
+
+    override fun getDomainLocale(): Locale {
+        return lazyDomainLocale
+    }
+
+    private companion object {
+        const val FALLBACK_DOMAIN = "nautilus.ink"
+        val DOMAIN_BY_LOCALE = mapOf(
```

Comment:
Let's make `val`'s in the object`private` as well

Reply:
Why though? The companion object is private.

Reply:
Sometimes it's needed to change visibility modifier of the companion object, so it's better to declare all variables intended to be private at once. But it's not critical, it's more like "just in case"